netlify.toml

```diff
@@ -9,7 +9,7 @@ command = "git submodule update --init --recursive --depth 1 && make non-product
 [build.environment]
 NODE_VERSION = "10.20.0"
 HUGO_VERSION = "0.101.0"
-RUBY_VERSION = "3.0.1"
+# RUBY_VERSION = "3.0.1"
```

Comment:
If you comment out the parameter, does it make sense to document the change in the file?